src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -1052,6 +1219,38 @@ def create_python_udf_or_sp(
         else ""
     )
 
+    # As an FYI, _should_continue_registration is a function, and is defined outside the Snowpark context.
+    if _should_continue_registration is None:
```

Comment:
If `should_continue_registraion` is only used in Native Apps Sandbox environment, what's the use case when this returns True?

Reply:
`_should_continue_registration` is not specific to a sandbox execution. This could be installed by any caller environment that just wants to inspect objects, e.g. testing, before they are created. So in that way, you could still want to continue registration.

Reply:
Yeah this is to make it more generic.